modules/userId/eids.js

```diff
@@ -45,6 +45,12 @@ export const USER_IDS_CONFIG = {
     atype: 1
   },
 
+  // pairId
+  'pairId': {
+    source: 'google.com',
+    atype: 571187
```

Comment:
where did you get this value?

Reply:
Hi Patrick, this is the atype value we picked to identify PAIR ID. Since PAIR is still in MVP phase and we only expect pubs/advertisers with their own user management systems to use this feature, at this moment the value is not documented publicly and mostly communicated directly with early adopter partners. I can confirm the value `571187` is correct and is the one will be used in google's ads serving system.
If you are aware of some centralized place for documenting atype, feel free to point it to me. happy to add it there.

Reply:
Hi @patmmccann, communicated with internal team, atype: 571187 is what we have been using in our codebase for PAIR ID.  I also checked the openRTB spec, 500+ are for Vendor-specific codes. So hopefully we should be fine using this value?

Reply:
this is the place https://github.com/InteractiveAdvertisingBureau/openrtb/blob/master/extensions/2.x_official_extensions/eids.md but your code has nothing to do with agent type, you appear to have hijacked an unrelated field and you should have used ext

Reply:
I thought you worked at liveramp, so above comment was supposed to be google or liveramp used the wrong value, but now it is clear just google made this choice.

Reply:
btw, I asked IABTL slack to weigh in on this question here https://openrtb-iabtechlab.slack.com/archives/C3Y6GHUTH/p1680635011100839 if you have access

Reply:
I think it depends on how do you interpret `agent type`, especially given that a person-based id (3) is considered an `agent type`, pairId is not a traditional person-based user id as its encrypted and protected combined publisher/adversiter 1p data. We are trying to use this to address various privacy concerns introduced by person-based ID. So we think it deserve a separate code.
Also, 'hijacked' is a strong word. again, given the specification that 500+ are for vendor-specific and we are only using the field for pairId, any secrutiy/privacy vulnerability you are aware of for using `atype` like this? If so, we will have to address that.

Reply:
I don't mean to imply any malintent with my word choice, just that you've put the value in the wrong place. Many other adapters have chosen to put their custom fields in ext.stype or something else, eg https://github.com/prebid/Prebid.js/blob/8c8a2d7d2aa9202ef0823d27ad7f5a079ac3c284/test/spec/modules/eids_spec.js#L64

If you PR IABTL with your choice and they accept it, I'm happy to merge as is. For now, it appears your value is misleading on agent type and you should have chosen ext.someotherkey

Reply:
As further background, vendor specific codes are typically enumerated to avoid conflicts, eg https://github.com/InteractiveAdvertisingBureau/openrtb/blob/aa2ce4cfdc568e54cc4b64d9fbdec52b1e3e2654/extensions/community_extensions/segtax.md?plain=1#L233

Reply:
Sounds good! Thank you for the pointers Patrick! I will discuss with internal team who own this feature and get back to this thread.